frontend/src/pages/chat/components/chat-layout/chat-layout.tsx

```diff
@@ -35,10 +39,20 @@ const ChatLayout: React.FC<Properties> = ({ filter }) => {
     });
   const hasId = Boolean(id);
 
+  const handleImageUpdate = useCallback(
+    (payload: UpdateChatImageRequestDto): void => {
+      void dispatch(chatActions.updateChatImage(payload));
+    },
+    [dispatch],
+  );
+
   const handleSend = useCallback(
     ({ message }: ChatInputValue): void => {
       if (!hasId || currentChatMessages.length === EMPTY_ARRAY_LENGTH) {
-        void dispatch(chatActions.createChat({ message }));
+        void dispatch(chatActions.createChat({ message })).then((action) => {
```

Comment:
```suggestion
        void dispatch(chatActions.createChat({ message })).unwrap().then((action) => {
```
In order to handle rejections automatically